neo/Ledger/Blockchain.cs

```diff
@@ -628,7 +606,7 @@ private void Persist(Block block)
                             Transaction = tx,
                             ExecutionResults = execution_results.ToArray()
                         };
-                        Distribute(application_executed);
+                        Context.System.EventStream.Publish(application_executed);
```

Comment:
Just a minor thing, the type of that was set was `PersistedCompleted`, will it also listen to `ApplicationExecuted`?

Reply:
No.

Reply:
Thus, who go this message `Context.System.EventStream.Publish(application_executed);`?